ecmascript/transforms/tests/modules_common_js.rs

```diff
@@ -4185,3 +4185,45 @@ function setup(url: string, obj: any) {
     return _url1;
 }"
 );
+
+test!(
+    ts_syntax(),
+    |_| tr(Config {
+        ..Default::default()
+    }),
+    lodash_es,
+    "import root from './_root.js';
+  import stubFalse from './stubFalse.js';
+  
+  var freeExports = typeof exports == 'object' && exports && !exports.nodeType && exports;
+
+  var freeModule = freeExports && typeof module == 'object' && module && !module.nodeType && \
+     module;
+  
+  var moduleExports = freeModule && freeModule.exports === freeExports;
+  
+  var Buffer = moduleExports ? root.Buffer : undefined;
+  
+  var nativeIsBuffer = Buffer ? Buffer.isBuffer : undefined;
+  
+  var isBuffer = nativeIsBuffer || stubFalse;
+  
+  export default isBuffer;
```

Comment:
Given this line, the file is treated as an es module so `exports` is just an identifier.

Reply:
As a side note: Renaming was intentional. See: https://github.com/babel/babel/blob/e498bee10f0123bb208baa228ce6417542a2c3c4/packages/babel-plugin-transform-modules-commonjs/src/index.js#L149-L153

Reply:
works fine in `babel + babel-plugin-transform-modules-commonjs`, [Babel repl](https://babeljs.io/en/repl#?browsers=defaults%2C%20not%20ie%2011%2C%20not%20ie_mob%2011&build=&builtIns=false&spec=false&loose=false&code_lz=JYWwDg9gTgLgBFCF4DNEjgcgHQHoD6iy2AVgM6YDcAUKJLHGTAK4BGAYgIYA2ZApnDQQMOXEzZdefUhRrUAbpyiCofPgFEAHvRhk4AXjgwAnmD4QUcPtui6DhzBFYk-AYxiY4AMi9WbsPR84AEJrHTJsADsIABM-ABVTASCw2zIaRWU0NQBZWOZuAUNsjX87IJMzCzgQfML7LCcXd08g2piC5N9g9s6o2ISk719qODHewrlMmrrS8IMVXNnhxb48jsLsVID7YtU5tKmlOAAhZhQUPmVDCYOdgH4EJBhsM4uruAAuOGZIuJRgJE-DEjspIpwYMB5HwAJJkN6Xa6nc6IuCPBFXbDAeEoj7fX7_QHA0FwbEYpHgyHQuHkuAAHzpjBYHB4_Dk23g_04BXgZNxUGoQA&debug=false&forceAllTransforms=false&shippedProposals=false&circleciRepo=&evaluate=false&fileSize=false&timeTravel=false&sourceType=module&lineWrap=true&presets=env%2Creact%2Cstage-3%2Cenv&prettier=false&targets=&version=7.11.1&externalPlugins=%40babel%2Fplugin-transform-modules-commonjs%407.10.4)